src/components/Header.tsx

```diff
@@ -136,12 +136,19 @@ const mainMenu: MenuEntry[] = [
         description: "Our user documentation and how-to guides.",
       },
       {
-        title: "Developer Resources",
-        url: { url: "https://docs.kodi.tv", type: "external" },
+        title: "Add-on Developer Resources",
```

Comment:
Is this an ordered list? If so, it seems odd that Addon docs are listed before the main application when addons are only extensions to the core app. I'd do it the other way around.

Btw, seems like docs.kodi.tv already points to the new location :)